bugbug/tools/code_review.py

```diff
@@ -595,10 +601,23 @@ def get_all_inline_comments(
                     continue
 
                 transaction_comment = transaction["comments"][0]
-                comment_id = transaction_comment["id"]
-                date_created = transaction_comment["dateCreated"]
                 comment_content = transaction_comment["content"]["raw"]
+                is_generated = (
+                    # This includes comments generated by Review Helper, but
+                    # excludes any comments that have been edited by the user.
+                    "> This comment was generated automatically and has been approved by"
```

Comment:
We must remember to keep this up to date in case we change the string

Reply:
I added a note referring to this in the add-on's backend